authutils/auth_service.go

```diff
@@ -22,6 +22,8 @@ import (
 
 const (
 	AllowedBlockChainDifference = 5
+	//1 month is approximately 172800 blocks
+	AllowedBlockDifferenceForAuthorizedToken = 172800
```

Comment:
We may pass it with token instead of hardcoding it.

Reply:
got away with this now , we just check if current block < tokens's Expiry block